core/src/main/java/com/pablisco/result/Result.java

```diff
@@ -17,16 +17,16 @@ public final boolean isSuccessful() {
         );
     }
 
-    public final S getSuccess() {
+    public final S getSuccessOrElse(S alternative) {
```

Comment:
Would it be handy for the Result class to have onSuccess or onFailure methods so user can react to the result?